### ISSUE TITLE
Settings should be closable in sub-pages with escape key

### ISSUE THREAD

**Issue by Simon-Laux** [deltachat/deltachat-desktop#3868]
Or go back first to main settings page and then close on the second press, I assume.

**Comment by r10s** [comment]
> and then close on the second press, I assume.

yip, the "Esc"-key should hit the topmost's dialog "back button"

and if if there is none, close the topmost dialog

**Comment by maxphilippov** [comment]
Is #2268 related? In terms of we need to manage how dialogs are stacked and which one is closed by ESC. I can take that after resolving more pressing bugs since I'm reworking Dialog as part of BlueprintJS removal

**Comment by friedmag** [comment]
I was looking at this as a first issue, and can confirm it's fixed by the commit above.

**Comment by Simon-Laux** [comment]
meanwhile this was solved in #4702
